index.bs

```diff
@@ -194,6 +214,28 @@ partial interface XRSession {
 };
 </pre>
 
+<div class="algorithm" data-algorithm="request-light-probe">
+When the <dfn method for="XRSession">requestLightProbe(|options|)</dfn> method is invoked on {{XRSession}} |session|, the user agent MUST run the following steps:
+  1. Let |promise| be [=a new Promise=].
+  1. If the [=light-estimation=] feature descriptor is not [=list/contain|contained=] in the |session|'s [=XRSession/list of enabled features=], [=/reject=] |promise| with {{NotSupportedError}} and abort these steps.
+  1. If |session|’s [=XRSession/ended=] value is <code>true</code>, throw an {{InvalidStateError}} and abort these steps.
+
+    <dl class="switch">
+      <dt>If |options|'s {{XRLightProbeInit/reflectionFormat}} is {{XRReflectionFormat/"srgba8"}}, matches {{XRSession/preferredReflectionFormat}} or is otherwise supported by the User Agent:</dt>
```

Comment:
I think we ONLY want to support `srgba8` or the `preferredReflectionFormat`, even if the system could potentially support other formats.

Reply:
Done.